db/migrate/20150611083448_change_custom_attribute_value_type_to_text.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeCustomAttributeValueTypeToText < ActiveRecord::Migration
+  def up
+    change_column :custom_attributes, :value, :text
```

Comment:
I don't follow, can you point me to that?

Values of the configurations are not limited to 255 chars, so we need more